### PR TITLE
Issue templates and picker minor grammar tweaks

### PR DESCRIPTION
**Pull request by kingthorin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/third-party-addon.yml
Comment (thc202) [inline .github/ISSUE_TEMPLATE/third-party-addon.yml:1]: `License` was not changed, on purpose?

Comment (kingthorin) [reply .github/ISSUE_TEMPLATE/third-party-addon.yml:1]: Nope missed it somehow, fix momentarily

**Comment by kingthorin** [comment]
Tweaked

**Comment by kingthorin** [comment]
Oops

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**